carma-messenger-config/development/docker-compose-background.yml

```diff
@@ -17,7 +17,7 @@ version: '2'
 
 services:
   carma-messenger-ui:
-    image: ${DOCKER_ORG}/carma-messenger-ui:${DOCKER_TAG}
+    image: usdotfhwastolcandidate/carma-messenger-ui:prowler
```

Comment:
same comment as before where we probably need to switch

Reply:
updated